### ISSUE TITLE
[Accessibility][Usable]: Narrator doesn't announce the slider value along with the control name while changing the 'Number of persona' slider using arrow keys. (Bug 4)

### ISSUE THREAD

**Issue by jdhuntington** [OfficeDev/office-ui-fabric-react#10858]
Tracking all of the issues brought up with the 1/4 accessibility pass on Facepile control 
Details on individual bugs can be found on [SharePoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7Bfd43fcc5-515d-47d8-946d-0d4862a56278%7D&action=default&uid=%7BFD43FCC5-515D-47D8-946D-0D4862A56278%7D&ListItemId=1647&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble) 

Breakout of issue #7534

**Comment by micahgodbolt** [comment]
issue was actually in slider itself. PR up
